src/service/auth/verifyCode.ts

```
@@ -0,0 +1,31 @@
+import redis from '../../config/redis';
+import { Request, Response } from 'express';
+import { VerifyCodeRequest } from '../../types/auth';
+import { BasicResponse } from '../../types';
+
+export const verifyCode = async (req: Request<{}, {}, VerifyCodeRequest>, res: Response<BasicResponse>) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)